rustls/Cargo.toml

```diff
@@ -47,7 +47,7 @@ log = { workspace = true, optional = true }
 # Using `once_cell::sync::OnceCell` would let the losing thread to see the winning value
 # in the error result from `crypto::CryptoProvider::set_default()`, which does not seem
 # worth using the `critical-section` feature with its no-std installation requirements.
-# TODO: specify `once_cell` version in single place - see also: https://github.com/rustls/rustls/pull/2350
+# TODO: specify `once_cell` version in single place - as RECENTLY UPDATED in upstream RUSTLS: https://github.com/rustls/rustls/pull/2350
```

Comment:
In general I am thinking to inherit the updated version number but NOT the features configuration from top-level `Cargo.toml`. I think it would be best to keep the ability to configurre the features llist as may be needed here moving forward.